jupyterhub/app.py

```diff
@@ -606,6 +606,31 @@ def _authenticator_default(self):
         """
     ).tag(config=True)
 
+    throttle_retry_suggest_min = Integer(
+        30,
+        help="""
+        Minimum seconds after which we suggest the user retry spawning.
+
+        When `concurrent_spawn_limit` is exceeded, we recommend users retry
+        after a random period of time, bounded by throttle_retry_suggest_min
+        and throttle_retry_suggest_max.
+
+        throttle_retry_suggest_min should ideally be set to the median
+        spawn time of servers in your installation.
+        """
+    )
+
+    throttle_retry_suggest_max = Integer(
+        60,
+        help="""
+        Minimum seconds after which we suggest the user retry spawning.
```

Comment:
Largest period (ceiling) in seconds to suggest a user wait before retrying.

When `concurrent_spawn_limit` is exceeded, spawning is throttled. We
suggest users wait a random period of time, bounded by the floor and ceiling,
before retrying.